pkg/network/ovn_kubernetes.go

```diff
@@ -416,10 +416,16 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	}
 	data.Data["ClusterManagerReplicas"] = clusterManagerReplicas
 
+	productFlavor := "self-hosted"
+	if bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.Enabled {
+		productFlavor = "managed"
+	}
+	manifestSubDirBasePath := filepath.Join(manifestDir, "network/ovn-kubernetes", productFlavor)
```

Comment:
this is a nit and don't need it on this PR

but isn't this supposed to be used as
```
manifestSubDirBasePath := filepath.Join(manifestDir, "network", "ovn-kubernetes", productFlavor)
```

Reply:
I guess its fine, ignore me

Reply:
Yeah, it really should be; I chose not to rework that bit to keep the changes more minimal.